connect/src/main/java/com/stripe/android/connect/EmbeddedComponentManager.kt

```diff
@@ -21,17 +35,31 @@ class EmbeddedComponentManager(
     private val fetchClientSecretCallback: FetchClientSecretCallback,
     appearance: Appearance = Appearance(),
     private val customFonts: List<CustomFontSource> = emptyList(),
+    private val logger: Logger = Logger.getInstance(enableLogging = BuildConfig.DEBUG),
+    private val isDebugBuild: Boolean = BuildConfig.DEBUG,
 ) {
     private val _appearanceFlow = MutableStateFlow(appearance)
     internal val appearanceFlow: StateFlow<Appearance> get() = _appearanceFlow.asStateFlow()
+    private val loggerTag = javaClass.simpleName
+
+    // Public functions
 
     /**
      * Create a new [AccountOnboardingView] for inclusion in the view hierarchy.
      */
+    @PrivateBetaConnectSDK
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
All these functions should not be restricted to the library -- they're part of the public API. Also, isn't `@PrivateBetaConnectSDK` redundant when it's already on the class?

Reply:
Yeah I didn't realize that this is only needed at the class level, so I cleaned it up throughout the library.

We're adding `@RestrictTo` until we're ready to go into private beta, at which point we'll remove it and external developers will be able to use these classes (with the right opt-in). See here: https://github.com/stripe/stripe-android/pull/9203#discussion_r1747690234